style.css

```diff
@@ -0,0 +1,317 @@
+        
+    }
+
+figure
+    {
+        text-align: center;
+    }
+
+.bio-menu
+    {
+        text-align: center;
+    }
+
+#bio-family-pics
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: space-evenly;
+        border-radius: 50%;
+    }
+
+#bio-teams-logos
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: space-evenly;
+
+    }
+
+#bio-travel-pics
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: space-evenly;
+    }
+
+#bio-pics
+    {
+        border-radius: 50%;
+    }
+
+.exp-menu
+    {
+        text-align: center;
+    }
+
+#exp-programs-pics
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: space-evenly;
+        align-items: center;
+    }
+
+#exp-editors-pics
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: space-evenly;
+        align-items: center;
+    }
+
+#exp-edu-school-list
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+    }
+
+#exp-edu-school-info
+    {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        align-content: center;
+    }
+
+#exp-edu-school-logo
+    {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        align-items: baseline;
+        text-align: center;
+    }
+
+#exp-awards
+    {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: first baseline;
+    }
+
+#exp-award-school
+    {
+        text-decoration: underline;
+        text-align: center;
+        font-weight: bold;
+    }
+
+.exp-degree
+    {
+        text-align: center;
+    }
+
+.port-menu
+    {
+        text-align: center;
+    }
+
+#port-console
+    {
+        display: flex;
+        flex-direction: row;
+    }
+
+#port-window
+    {
+        display: flex;
+        flex-direction: row;
+    }
+
+#port-window1
+    {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+#port-window2
+    {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+#port-unity
+    {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+#port-unity1
+    {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+.combat
+    {
+        text-align: center;
+    }
+
+.guess
+    {
+        text-align: center;
+    }
+
+.helper
+    {
```

Comment:
Is there a way we could combine all of these "text align centers" into one CSS query?